tests/unit/constraints/test_tabular.py

```diff
@@ -1802,6 +1802,36 @@ def test_transform(self):
         })
         pd.testing.assert_frame_equal(expected_out, out)
 
+    def test_transform_without_dropping_column(self):
+        """Test the ``ColumnFormula.transform`` method without dropping the column.
+
+        If `drop_column` is false, expect to not drop the constraint column.
+
+        Input:
+        - Table data (pandas.DataFrame)
+        Output:
+        - Table data without the indicated column (pandas.DataFrame)
```

Comment:
This should say "with the indicated column" right? Since it isn't being dropped

Reply:
Yes, thanks!